src/completion-provider.js

```diff
@@ -110,6 +116,28 @@ class CompletionProvider {
       (suggestions) => this.filterSuggestions(prefix, suggestions)
     );
   }
+
+  lookupWebpack(prefix) {
+    const projectPath = atom.project.getPaths()[0];
+    if (!projectPath) {
+      return Promise.resolve([]);
+    }
+
+    const webpackConfigFilename = atom.config.get('autocomplete-modules.webpackConfigFilename');
+    const webpackConfigPath = path.join(projectPath, webpackConfigFilename);
+    const vendors = atom.config.get('autocomplete-modules.vendors');
+
+    const webpackConfig = require(webpackConfigPath);
+    let moduleSearchPaths = get(webpackConfig, 'resolve.modulesDirectories', []);
```

Comment:
The webpack config should always just be a plain object but I can appreciate the concern. It seems silly, but I try to avoid try/catch where practical. The try/catch seems overkill for a plain key/value object. I'll look at some options here.